frontend/src/routes/about/us/+page.svelte

```diff
@@ -70,58 +58,58 @@
 						)}
 						alt="Profilbild"
 					/>
-					<div class="flex flex-col gap-6 p-4">
-						<h3 class="text-base font-bold">{statement.subject}</h3>
-						<h2 class="text-2xl font-bold uppercase">{statement.name}</h2>
+					<div class="flex flex-col gap-2 p-4 sm:gap-8">
+						<b class="h-4">{statement.subject}</b>
+						<h2>{statement.name}</h2>
 						<p>{statement.statement}</p>
 					</div>
 				</div>
 			{/each}
 		</div>
-	</div>
+	</section>
 
-	<div class="flex flex-col gap-8 px-4 py-20 xl:px-80">
-		<h1 class="break-words text-4xl font-bold uppercase md:text-5xl">
-			Hochschul-SMD <br /> Ein deutschlandweites Netzwerk
-		</h1>
-		<div>
-			Als Hochschulgruppe der SMD sind wir eine ehrenamtliche Initiative. Unterstützt werden wir von
-			einer hauptamtlichen Regionalreferentin und ehrenamtlichen Gruppenbegleitern (SMDler, die ihr
-			Studium schon hinter sich gebracht haben.) Wenn du wissen möchtest, wie die Hochschul-SMD
-			überregional aufgebaut ist, schau dir <a
-				class="text-primary"
-				href="https://www.smd.org/hochschul-smd/hochschul-smd/herzlich-willkommen/"
-			>
-				diese Seite
-			</a> an.
-		</div>
-	</div>
+	<!-- TODO add Leiter Block -->
 
-	<div class="flex flex-col gap-8 px-4 xl:px-80">
-		<h1 class="text-4xl font-bold uppercase md:text-5xl">Leitungsteam</h1>
+	<div class="flex flex-col gap-8 px-4 py-20">
+		<h1 class="text-center">Unser Netzwerk</h1>
+		<div>
+			<img
+				class="pb-6 max-sm:w-full sm:float-right sm:w-[40] sm:max-w-sm xl:max-w-lg"
+				src={regions}
+				alt="Regionen"
+			/>
+			<p class="pt-4">
+				Die SMD ist ein deutschlandweites Netzwerk christlicher Hochschulgruppen. Wir sind eine von
+				ca. 80 anderen SMD-Gruppen in ganz Deutschland. Uns gibt es in fast jeder Univeristätsstadt.
+				Wir lieben es auch uns überregional zu vernetzen und uns auszutauschen. Dafür gibt es zwei
```

Comment:
```suggestion
				Wir lieben es auch, uns überregional zu vernetzen und auszutauschen. Dafür gibt es zwei
```
* comma because of [D125, point 3](https://www.duden.de/sprachwissen/rechtschreibregeln/komma#D125) (I only added this link because I needed to look it up myself to be sure that this is non-optional.)
* leaving `uns` out sounds more fluent to me, but your version should be correct as well